nemo/core/classes/mixins/adapter_mixins.py

```diff
@@ -150,6 +150,20 @@ class AdapterModuleMixin(ABC):
     adapter_global_cfg_key = "global_cfg"
     adapter_metadata_cfg_key = "adapter_meta_cfg"
 
+    def set_accepted_adapters(self, adapters):
+        self.accepted_adapters = adapters
+
+    def get_accepted_adapters(self,):
+        if hasattr(self, 'accepted_adapters'):
+            return self.accepted_adapters
+        else:
+            return []
+
+    def get_from_adapter_layer(self, name: str):
```

Comment:
No need for this method. ModuleDict has get(str, default) semantics, use that.

Reply:
:) I thought the same, but turns out ModuleDict does not have `get(str, default)`